### PR TITLE
Move hot procs to CODEOWNERS for dev cycles initiative

### PR DESCRIPTION
**Pull request by Mothblocks**
Someone touched these recently and added seconds of init time, I don't want it to happen again without me knowing because it is hard to remove after the fact without just killing the entire features that rely on it

### CODE REVIEW

**Comment by Cyberboss** [comment]
Pretty sure `**/*_EXPENSIVE` will work if they are all just going to you two.

**Comment by LemonInTheDark** [comment]
It sucks that we need to pull them out like this, but I get why so yeah go off

File: code/game/turfs/open/space/space_EXPENSIVE.dm [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		if(requires_activation)
+			SSair.add_to_active(src, TRUE)
+
+		if(SSmapping.max_plane_offset)
+			var/turf/T = GET_TURF_ABOVE(src)
+			if(T)
+				T.multiz_turf_new(src, DOWN)
+			T = GET_TURF_BELOW(src)
+			if(T)
+				T.multiz_turf_new(src, UP)
```
[the remark this answers is not in the document]
Comment (Mothblocks) [reply code/game/turfs/open/space/space_EXPENSIVE.dm:45]: This is just moving code over, we shouldn't do anything without profiling. I can show you how to use Tracy and the strategies to test stuff like this though 🙂

[the remark this answers is not in the document]
Comment (Ghommie) [reply code/game/turfs/open/space/space_EXPENSIVE.dm:45]: Yea, sorry. I should have specified I was simply pointing out.

**Comment by LemonInTheDark** [comment]
it would need to include subtypes for that, and that is just silly.
it would be nicer to in place mark procs with like an annotation or something but that's not happening soon

**Pull request merged**